.gitignore

```diff
@@ -90,6 +90,7 @@ saml_*.key
 /public/assets
 /public/packs
 /app/assets/builds/*
+/serialized
```

Comment:
where is this path referenced in the code?

Reply:
it is a folder generated by a script in identity-idp-config that has to be copied over to run the test, I just want to make sure it doesn't accidentally get committed if someone copies it over

Reply:
Could we have that script write to `./tmp` or something? that would also prevent accidental commits since we ignore that already?

Reply:
Sure I can do that